### DISCUSSION TITLE
bluetooth password

### DISCUSSION THREAD

**Discussion by BOPOHOP** [Patrick762/hassio-bluetti-bt#146] [General]
Bluetooth connection to some  bluetti devices (for example  EP500Pro) can be password protected (this can be turned on in application). Do you have plans to supports password protected BT?

**Comment by LeoAlioth** [comment]
There is already progress being made, and it is discussed here:

https://github.com/Patrick762/hassio-bluetti-bt/issues/120#issuecomment-2629313227
